keps/sig-cluster-lifecycle/0015-kubeadm-join-master.md

```diff
@@ -0,0 +1,431 @@
+# kubeadm join --master workflow
+
+## Metadata
+
+```yaml
+---
+kep-number: 15
+title: kubeadm join --master workflow
+status: accepted
+authors:
+  - "@fabriziopandini"
+owning-sig: sig-cluster-lifecycle
+reviewers:
+  - "@cha”
+  - "@jdtibier"
+approvers:
+  - "@luxas"
+  - "@timothysc"
+editor:
+  - "@fabriziopandini"
+creation-date: 2018-01-28
+last-updated: 2018-06-29
+see-also:
+  - KEP 0004
+```
+
+## Table of Contents
+
+<!-- TOC -->
+
+- [kubeadm join --master workflow](#kubeadm-join---master-workflow)
+    - [Metadata](#metadata)
+    - [Table of Contents](#table-of-contents)
+    - [Summary](#summary)
+    - [Motivation](#motivation)
+        - [Goals](#goals)
+        - [Non-goals](#non-goals)
```

Comment:
can you add prerequisites?

Reply:
Prerequisites is not part of the KEP template.. What do you foresee in this chapter? all the prerequisites for this specific proposal are already described in the Implementation Details paragraph ...